Src/library/base/elks/kernel/function.e

```diff
@@ -40,16 +41,8 @@ feature -- Access
 			-- Result of calling function with `args' as operands.
 		require
 			valid_operands: valid_operands (args)
-		local
-			l_closed_count: INTEGER
-			c: like closed_operands
-		do
-			c := closed_operands
-			if c/= Void then
-				l_closed_count :=  c.count
-			end
-			Result := fast_item (encaps_rout_disp, calc_rout_addr, $closed_operands, $args, routine_id,
-				is_basic, written_type_id_inline_agent, l_closed_count, open_count, $open_map)
+		external
+			"built_in"
```

Comment:
The call to 'fast_item' is too specific to the ISE compiler, hence the addition of these two built-in features. Furthermore, the fact that 'item' is built-in allows the Gobo compiler to optimize the code when the argument is a manifest tuple.